0.10/contrib/etc/generate_container_user

```diff
@@ -0,0 +1,19 @@
+# Set current user in nss_wrapper
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+
+if [ x"$USER_ID" != x"0" -a x"$USER_ID" != x"1001" ]; then
+
+    NSS_WRAPPER_PASSWD=/opt/app-root/etc/passwd
+    NSS_WRAPPER_GROUP=/etc/group
+
+    cat /etc/passwd | sed -e 's/^default:/builder:/' > $NSS_WRAPPER_PASSWD
```

Comment:
user 1001 exists in our /etc/passwd file because sti-base provided it, but really that user is the "builder" user (it's what assemble runs as).  

Since we need to create a new default user to represent the random id this container is running as, we first need to rename the old "default" user to something else so there is no conflict (we could also just filter it out entirely) and renaming it to builder seems logical.

this actually came from @GrahamDumpleton's suggested fix to the python image.

feline removed.